src/bda/plone/shop/browser/toolbar.py

```diff
@@ -51,6 +51,9 @@ def getMenuItems(self, context, request):
         navigation = ShopNavigation(context, request)
         items = list()
         for group in navigation.groups():
+            group_links = [x for x in group.links()]
```

Comment:
can you use ``group_links`` in the loop below instead of calling ``group.links()`` again please?